.github/workflows/nightly-ci.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ ubuntu-latest, macos-latest, windows-latest ]
-        java-version: [ 11, 17, 19 ]
+        java-version: [ 8, 11, 17, 19 ]
```

Comment:
Does it worked expectly?

Reply:
I see, JDK 8 will not compile correctly. I will remove this.